config/input-resource-verify.xml

```diff
@@ -0,0 +1,44 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<project name="input-validate" default="execute">
+
+  <condition property="checkstyle.ant.skip">
+    <isset property="checkstyle.ant.skip" />
+  </condition>
+
+  <target name="execute" unless="checkstyle.ant.skip">
+    <taskdef name="checkstyle"
+             classname="com.puppycrawl.tools.checkstyle.ant.CheckstyleAntTask"
+             classpath="${mvn.runtime_classpath}"
+             />
+
+    <property name="checkstyle.pattern.todo" value="NOTHingWillMatCH_-"/>
+    <property name="check.config" location="config/checkstyle_resource_checks.xml"/>
```

Comment:
@romani I specifically wanted it to be a general name as I think we should add other checks to the configuration besides `NewlineAtEndOfFile`.

The 2 I would like to add is: package name should be same as folder location, class name should be the same as file name.
Some others that have confused new users are tabs and trailing spaces.